client/src/components/Query/Filter/index.jsx

```diff
@@ -238,6 +238,7 @@ Filter.defaultProps = {
   onEditCallback: () => {},
   onRemoveCallback: () => {},
   onSelectCallback: () => {},
+  label: null,
```

Comment:
@claudia1296 Probably should be `''` empty string since we're defining the shape like ` label: PropTypes.string`